src/Compilers/CSharp/Portable/Syntax/InternalSyntax/TypeSyntax.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax
+{
+    internal abstract partial class TypeSyntax
+    {
+        public bool IsVar
```

Comment:
Also, there is an `IsVar` extension method in `src\Compilers\CSharp\Portable\Syntax\SyntaxFacts.cs`. Could it be used instead?

---
In reply to: [90152557](https://github.com/dotnet/roslyn/pull/15548#discussion_r90152557) [](ancestors = 90152557)